mesonbuild/interpreter/type_checking.py

```diff
@@ -174,6 +174,13 @@ def variables_convertor(contents: T.Union[str, T.List[str], T.Dict[str, str]]) -
     convertor=_install_mode_convertor,
 )
 
+INSTALL_NAME_KW: KwargInfo[T.Optional[str]] = KwargInfo(
+    'install_name',
+    (str, NoneType),
+    default=None,
+    since='1.2.0',
```

Comment:
This should be 1.3.0 since the window for features for 1.2.0 is over at this point. Exciting feature!